examples/tutorial/metric-learning.ipynb

```diff
@@ -0,0 +1,668 @@
+{
```

Comment:
<p>remove double quotation between learning and tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/objax/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>